### PR TITLE
Fixed broken master

### PR DESCRIPTION
**Pull request by Nyholm**
I noticed master was broken due old dependencies. 

I removed dependencies not used and updated minor versions on existing dependencies. Composer will run way quicker now. =)

### CODE REVIEW

**Comment by Nyholm** [comment]
I noticed that we have not built code coverage since 2014. (https://coveralls.io/github/backup-manager/backup-manager?branch=master). I added this here again. 

Btw, we should configure travis to run updates every day/week.

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,16 @@
 language: php
 
-sudo: false
+os: linux
```
Comment (Nyholm) [inline .travis.yml]: From Travis: 
Screenshot 2020-02-23 at 16 49 54

**Comment by Nyholm** [comment]
@ShawnMcCool I know you are hovering the merge button at this moment. 

I want to encourage you to press it ?=

**Comment by ShawnMcCool** [comment]
Ah sorry, I didn't see this.

**Comment by Nyholm** [comment]
Thank you for merging

=)

**Pull request merged**